suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx

```diff
@@ -96,7 +96,11 @@ export const NetworkSymbolSwitchItem = ({
     };
 
     return (
-        <Pressable onPress={_ => handleEnabledChange(!isEnabled)} accessibilityRole="togglebutton">
+        <TouchableOpacity
+            onPress={_ => handleEnabledChange(!isEnabled)}
+            accessibilityRole="togglebutton"
+            activeOpacity={0.6}
```

Comment:
Why overriding default opacity value?

Reply:
@shenkys mentioned before (in add new coin), that default 0.2 is too much transparent.